react/features/filmstrip/components/Filmstrip.native.js

```diff
@@ -86,10 +92,28 @@ class Filmstrip extends Component<Props> {
                         /* eslint-enable react/jsx-wrap-multilines */
                     }
                 </ScrollView>
+                {
+                    isNarrowAspectRatio_
+                    && this._renderLocalThumbnail()
+                }
             </Container>
         );
     }
 
+    /**
+     * Renders the local thumbnail.
+     *
+     * @private
+     * @returns {ReactElement}
+     */
+    _renderLocalThumbnail() {
+        return (
+            <View style = { styles.localThumbnail }>
```

Comment:
since you're adding a dedicated LocalThumbnail component maybe we don't need the _renderLocalThumbnail method, but just move the styling there and in the code above just use <LocalThumbnail> directly ?

Reply:
Good point, fixed, thanks!